free-courses-en.md

```diff
@@ -79,11 +79,13 @@
 
 ### Clojure
 
+* [BraveClojure](http://www.braveclojure.com)
```

Comment:
Direct link to the book: `http://www.braveclojure.com/clojure-for-the-brave-and-true/` ?